code/game/dna/genes/monkey.dm

```diff
@@ -83,6 +83,7 @@
 		I.implanted = O
 //		O.update_icon = 1	//queue a full icon update at next life() call
 	qdel(M)
+	O.active_genes |= /datum/dna/gene/monkey //For some reason, this isn't done otherwise

```

Comment:
Now if you want something to complain about, it's this right here. There's something weird going on with the whole domutation2 loop that I couldn't figure out, and this is the (terrible) work-around.